src/__experimental__/components/checkable-input/checkable-input.style.js

```diff
@@ -18,14 +18,12 @@ const StyledCheckableInputWrapper = styled.div`
     disabled, fieldHelpInline, inputWidth, labelAlign, labelWidth, theme
   }) => css`
     ${FormFieldStyle} {
-      display: flex;
-      flex-wrap: wrap;
+      display: block;
```

Comment:
If I remember correctly - this display type was set to `flex` to properly align the input with fieldhelp and label. Are you sure that it should be changed to `block`?

Reply:
my investigation and testing concluded that flex was inappropriately used in this implementation together with white-space: nowrap;
I tested help button in both Checkbox and Switch components to keep styling as it was but resolve overflow issue that was caused by white-space: nowrap; , having flex was limiting the width of label component and breaking sentence into multiple lines and white-space: nowrap was fixing that line breaking.